### PR TITLE
part of osra339: adds 'New Sponsor' link to Sponsor#index view +spec

### PR DESCRIPTION
**Pull request by razorcd**
related to:
https://osraav.atlassian.net/browse/OSRA-339

### CODE REVIEW

**Comment by theB264** [comment]
OSRA339
https://osraav.atlassian.net/browse/OSRA-339

File: app/views/hq/sponsors/index.html.haml [hunk 1]
```diff
@@ -1 +1,5 @@
+.action_items
+  .class
```
Comment (NikitaAvvakumov) [inline app/views/hq/sponsors/index.html.haml]: We have a div whose class is "class"?

Comment (theB264) [inline app/views/hq/sponsors/index.html.haml]: I think he's referring to class action items as opposed to instance or application action items.  This naming, however awkward, is consistent with the other views

Comment (NikitaAvvakumov) [inline app/views/hq/sponsors/index.html.haml]: Let's change it everywhere! :)

Comment (theB264) [inline app/views/hq/sponsors/index.html.haml]: :+1:

**Comment by razorcd** [comment]
I removed `.class` completely, I don't see any reason for having a class there

**Comment by NikitaAvvakumov** [comment]
Needs rebasing

**Pull request merged**